openquake/server/views.py

```diff
@@ -787,9 +787,8 @@ def aristotle_run(request):
     for key in ['dip', 'strike']:
         if key in rupdic and rupdic[key] is None:
             del rupdic[key]
-    user = utils.get_user(request)
     [jobctx] = engine.create_jobs(
-        [params], config.distribution.log_level, None, user, None)
+        [params], config.distribution.log_level)
```

Comment:
If we don't pass the currently logged in user, the ownership of the calculation will not be correct.